tests/unit/spec/base/mozilla-traffic-cop.js

```diff
@@ -118,6 +118,54 @@ describe('mozilla-traffic-cop.js', function () {
 
             expect(cop.verifyConfig()).toBeFalsy();
         });
+
+        it('should return true with a valid cookieExpires value', function() {
+            var cop = new Mozilla.TrafficCop({
+                id: 'test123',
+                cookieExpires: 48,
+                variations: {
+                    'v=1': 20,
+                    'v=2': 40
+                }
+            });
+
+            expect(cop.verifyConfig()).toBeTruthy();
+        });
+
+        it('should return false with an invalid cookieExpires value', function() {
+            var cop = new Mozilla.TrafficCop({
+                id: 'test123',
+                cookieExpires: '2 days',
+                variations: {
+                    'v=1': 20,
+                    'v=2': 40
+                }
+            });
+
+            expect(cop.verifyConfig()).toBeFalsy();
+        });
+    });
+
+    describe('Mozilla.TrafficCop.cookieExpiresDate', function() {
+        var config = {
+            id: 'test123',
+            variations: {
+                'v=1': 40,
+                'v=2': 30
+            }
+        };
+
+        it('should return null when no hours are specified', function() {
+            var cop = new Mozilla.TrafficCop(config);
+            expect(cop.cookieExpiresDate()).toBeFalsy();
```

Comment:
Would it be better to assert `null` here? A falsy value could also be `0` for example, which could result in a cookie that expired immediately?